training/README.rst

```diff
@@ -0,0 +1,163 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+========
+Training
+========
+
+This module was written to extend the functionality of events to support
+trainings and allow you to manage training actions, training groups,
+training hour types, training materials, etc.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Have *Training / User* or *Training / Manager* permissions.
+* Go to the new menu item called *Training*.
+
+The *Training* menu
+-------------------
+
+There you can find the following menus, depending on your permissions:
+
+Actions
+~~~~~~~
+
+Training actions define the structure of a course.
+
+They have a name, such as *Prevention of occupational hazards, Office
+software, Odoo user basic training*, etc.
+
+They are classified with action types. Depending on it, they define how many
+hours of each type are expected to be teached.
+
+You can define the training materials that should be delivered to students to
+accomplish the action.
+
+You can specify the grade settings: which is the minimal and maximum grade
+possible for this course, and which grade students need to reach to pass the
+course.
+
+Also you can define the contents of this course. They will appear in the back
+side of the diploma if available. You have a field called *Append template*
+that will help you to fulfill the contents in some complex layouts.
+
+Action Types
+~~~~~~~~~~~~
+
+Depending on the type of hours that a training action is expected to have, the
+training action has that action type. For instance:
```

Comment:
Training action has the same action type than the type of hours of a training action.